Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx

```diff
@@ -199,6 +203,7 @@ export function CreateOptionsV2(props: CreateOptionsProps) {
 
   useEffect(() => {
     if (currentTemplate) {
+      setReadMe('');
       props.fetchReadMe(currentTemplate);
     }
   }, [currentTemplate]);
```

Comment:
it's better if you add props.fetchReadMe to the dependencies list

Reply:
Not sure what you mean here props.fetchReadMe is in the dependency list. Do you mean props.setReadMe should be in the dependency list and passed from the parent? If so I have made that change. 

Going to keep this comment unresolved until I validate I correctly understand the original comment.

Reply:
Soroush, clarified on Teams and I have made the desired change to add to the useEffect dependency list